### DISCUSSION TITLE
How is this package different from Node YTDL-Core?

### DISCUSSION THREAD

**Discussion by twicer-is-coder** [LuanRT/YouTube.js#694] [General]
A basic comparsion and how both serves different purpose.

**Comment by ChunkyProgrammer** [comment]
Some differences:
- this package still receives regular updates
- this package supports more than just the watch page (ex: search, channels, trending, comments, playlists)
- this package probably has better typings since it's written in typescript while ytdl-core has a types definition file that they have to maintain and might be out of date
- this package can be more complex to use since it does a lot more thing

Also, I think ytdl-core uses html scraping which is more likely to get ratelimited. This package uses InnerTube which is an internal youtube API that is usually faster than html scraping.
